### PR TITLE
Core module angular

### PR DESCRIPTION
**Pull request by profileEnumerable**

### CODE REVIEW

File: Src/Campus.Master.API/UI/src/app/core/services/sign-in.service.spec.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { SignInService } from './sign-in.service';
 import { TestBed } from '@angular/core/testing';
 import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
-import { AuthenticatedUser } from '../modules/auth/shared/models/authenticated-user';
+import { AuthenticatedUser } from '../../modules/auth/shared/models/authenticated-user';
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/core/services/sign-in.service.spec.ts:4]: You can aggregate model imports in the `index.ts` to minimize directives

File: Src/Campus.Master.API/UI/src/app/core/services/sign-in.service.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
-import { AuthenticatedUser } from '../modules/auth/shared/models/authenticated-user';
+import { AuthenticatedUser } from '../../modules/auth/shared/models/authenticated-user';
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/core/services/sign-in.service.ts:3]: You can aggregate model imports in the `index.ts` to minimize directives

File: Src/Campus.Master.API/UI/src/app/modules/auth/registration/registration.component.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ import { HttpErrorResponse } from '@angular/common/http';
 import { Subscription, zip } from 'rxjs';
 import { RegisterUser } from '../shared/models/register-user';
 import { StateTransfer } from '@shared/models/state-transfer';
-import { LocaleService } from '../../../shared/services';
+import { LocaleService } from '../../../core/services/locale.service';
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/modules/auth/registration/registration.component.ts:15]: You can aggregate services imports in the `index.ts` to minimize directives

File: Src/Campus.Master.API/UI/src/app/modules/auth/sing-in/sing-in.component.ts [hunk 4]
```diff
@@ -1,12 +1,12 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { FormControl, FormGroup, Validators } from '@angular/forms';
-import { SignInService } from '@core/sign-in.service';
+import { SignInService } from '../../../core/services/sign-in.service';
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/modules/auth/sing-in/sing-in.component.ts:3]: You can aggregate services imports in the `index.ts` to minimize directives

File: Src/Campus.Master.API/UI/src/app/core/core.module.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      SignInService,
+      LocaleService,
+      TranslateStore,
+      {
+        provide: HTTP_INTERCEPTORS,
+        useClass: HttpTokenInterceptor,
+        multi: true
+      }],
+  imports: [
+    TranslateModule.forChild({
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/core/core.module.ts:24]: In Core module it's better to use `forRoot()` configuration. Here is an [example](https://github.com/ngx-translate/core#1-import-the-translatemodule)

**Review by antonAce — APPROVED** [review]
:shipit: with suggestions

File: Src/Campus.Master.API/UI/src/app/core/core.module.ts (hunk 5 above)
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/core/core.module.ts]: Replaced

**Pull request merged**